### PR TITLE
TINY-8594: Fixed inline toolbar flickering when switching between editors

### PR DESCRIPTION
**Pull request by tiny-ben-tran**
Related Ticket: TINY-8594, also fixed TINY-8503

Description of Changes:
* Reverted changes that were made in #6916 

The issue occurring in a single frame makes it difficult to write a reliable test

Pre-checks:
* [x] Changelog entry added
* [x] Tests have been added (if applicable)
* [x] Branch prefixed with `feature/` for new features (if applicable)

Review:
* [ ] Milestone set
* [ ] Review comments resolved

GitHub issues (if applicable):

### CODE REVIEW

File: modules/tinymce/src/themes/silver/main/ts/modes/Inline.ts [hunk 1]
```diff
@@ -109,19 +108,18 @@ const render = (editor: Editor, uiComponents: RenderUiComponents, rawUiConfig: R
 
   // In certain circumstances we need to delay the render function until the next
   // event loop to ensure things like the current selection are correct.
```
Comment (TheSpyder) [inline modules/tinymce/src/themes/silver/main/ts/modes/Inline.ts:110]:
```suggestion
```

**Comment by TheSpyder** [comment]
In Ben's testing, and I have confirmed, it doesn't regress the original issue. I'm guessing some other change in TinyMCE 6 resolves it.

**Comment by HAFRMO** [comment]
Excellent.

**Review by HAFRMO — APPROVED** [review]

**Comment by tiny-ben-tran** [comment]
Closed in favor of PR #7747, which has the correct branch name and is based on the correct branch `master`

**Pull request closed without merging**